app/src/main/java/com/partymusicq/android/ui/HomeActivity.kt

```diff
@@ -97,13 +104,16 @@ class HomeActivity : BaseActivity(), PartyAdapter.OnQueueSelectedListener {
 
     override fun onQueueSelected(queue: DocumentSnapshot) {
         val party = queue.toObject(Party::class.java)
-        if(party == null) {
+        if(party != null) {
```

Comment:
Could we combine this if statement with the one inside of it?

Reply:
yeah i can actually get rid of the party null check with a safe call and check if the id is null